### PR TITLE
Added Block Producer Rewards Calculator in Tools

### PR DESCRIPTION
**Pull request by fluciotto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ List of content:
 * [MinaExplorer API](https://graphql.minaexplorer.com/) Open GraphQL API from MinaExplorer
 * [Validators List](https://mina.staketab.com/) Overview of validators for mainnet
 * [MinaGraph API](https://minagraph.com) An alternative public GraphQL API node for Mina, exposed from the original GraphQL API of the Mina daemon
+* [Block Producer Rewards Calculator](https://towerstake.com/block-producer-rewards-calculator/) Calculate an expected return from running a validator node on Mina Protocol
```
Comment (nerdvibe) [inline README.md:47]: In general, we don't directly to the user, we add a description of the tool. Making an exception would sound odd imo. 

I would suggest changing this to: `Calculator for estimating the returns as a validator`

**Review by nerdvibe — CHANGES_REQUESTED** [review]
Hey @fluciotto thank you for your submission 🚀  The entry sounds good, I just kindly ask you to change a bit the description :)

File: README.md (hunk 1 above)
Comment (fluciotto) [reply README.md:47]: Hi,
Thanks for the advice, i change the description.

**Comment by nerdvibe** [comment]
Thank you 🙏 merging now 🥁

**Pull request merged**